### ISSUE TITLE
Questions about the data usage in Multi-task Pretraining stage

### ISSUE THREAD

**Issue by yuezewang** [QwenLM/Qwen-VL#89]
Hi, thanks for your amazing work! May I ask you the insight or perspective about the usage of image-text interleaved datasets (rather than manually packed)? In other words, do you consider using naturally interleaved image-text data? Or, there is a problem in the usage of them? Looking forward to your concern and reply, thanks~

**Comment by ShuaiBai623** [comment]
Yes, manually packed data is used to better train the model for few-shot learning. Of course, natural interleaved image-text data is also necessary, especially for image-text association and image-image understanding. We are also exploring this aspect.
